<launch>
        <node pkg="ros_monitoring" type="cpu_monitoring_node" name="cpu_monitor" output="screen" launch-prefix="chrt -f 50" respawn="true">
		<rosparam command="load" file="$(find ros_monitoring)/config/cpumonitorconfig.yaml" />
	
	</node>
</launch>
